targets/DeployDependencies.proj

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="DeployDependencies" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+
+  <Target Name="CopyPackageContent">
```

Comment:
I think you can avoid this double-hop indirection.  I sent you https://github.com/dsplaisted/msbuild/pull/1.